### DISCUSSION TITLE
Surrogate model of divertor detachment

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#2945] [Ideas]
In GitLab by @mkovari on May 5, 2022, 16:35

Stuart Henderson has improved on a simple surrogate model of divertor detachment originally created by Arne Kallenbach.  (Not to be confused with Arne's 1D divertor model which was the basis for the surrogate model.)  Why put this in PROCESS when we already have the 1D model?  There a few reasons:
- The 1D model has some numerical problems which can cause convergence difficulties.
- Stuart's model has been benchmarked against SOLPS and experiments.

The new model is [Scaling of divertor detachment to power and machine size](https://tdms.ukaea.uk/TDMS/DocumentWriter?ID=TD-0011993&REV=LATEST).  
See also [Technology R&D strategy: Primary divertor design](https://tdms.ukaea.uk/TDMS/DocumentWriter?ID=TD-0013706&REV=LATEST)

Any views @stuartmuldrew @ajpearcey ?

**Inputs**  

Neutral pressure in the divertor region, $`p_0`$.  Stuart H proposes that we fix this at 10 Pa.

Separatrix temperature: $`T_{𝑒,𝑠𝑒𝑝}`$

Core Zeff: $`Z_{eff}`$

Core Ar concentration: $`c_{𝐴𝑟}^{core}`$

Power crossing the separatrix: $`P_{sep}`$

Power fraction towards divertor in question: $`f_{SOL}`$

Major radius of divertor: $`R^{div}`$

Poloidal magnetic field at midplane: $`B_{\theta,mp}`$ 

Argon enrichment factor

**COMMENTS**

Detachment: This formula requires $`\lambda_{int}`$ to account for the radial diffusion power spreading, therefore using $`\lambda_{q}^{mp}`$ in the equation below represents a worst-case scenario.  Stuart has used lambda_int using the equation for S in the report.
```math
DP=1.3 \frac {R_0}{R^{mp}}  \times \frac {P_{sep}f_{SOL}}{R_{div}\mathrm{\ MW m^{-1}}}  \times  \frac {5  \mathrm{\ mm}}{\lambda_{int}}     \times {\left( \frac {R^{div}}{1.65\mathrm{\ m}} \right)}^{0.11}
```

Stuart H is planning to add further factors to the equation for DP. TODO

$`f_Z`$ is the relative radiation efficiency of Ar compared to D.  $`f_Z=90`$ at low divertor temperature and $`f_Z<<90`$ when the divertor attaches.  Since we don't have a formula for $`f_Z`$ when the divertor is partially attached, we need to make sure the divertor is detached, so the ratio in the equation for peak heat load must be > 1:

```math
f_Z=90
```
```math
DP<(1+f_Z c_{𝐴𝑟}^{div})p_0  
```

Peak target heat flux:
```math
q_\parallel^{div,peak} = 10 \mathrm{\ MW m^{-2}} \left(\frac {DP}{(1+f_Z c_{𝐴𝑟}^{div})p_0}\right)^{1.2}
```

Power fall-off length at midplane.
```math
\lambda_{q}^{mp} = 0.63 \ {\mathrm{mm}} \ B_{\theta,mp}^{-1.19}
```

Divertor Ar concentration:  This equation gives divertor argon concentration in terms of the puffing rates for fuel and argon, but we won't use it since PROCESS doesn't use puffing rates.  Instead we will just guess the argon enrichment in the divertor zone:
```math
c_{𝐴𝑟}^{div}=\mathrm{enrichment \ factor}\times c_{𝐴𝑟}^{core}
```

Divertor pressure: Stuart assumes that the neutral pressure in the divertor region $`p_0`$ is proportional to the fuelling rate $`\Gamma_D`$.  This is not logically consistent with Arne Kallenbach's 1 D model, which assumes 100% recycling and no fuelling.  I am not sure whether it makes sense to ignore this.

Separatrix density (equation 6):
```math
𝑛_{𝑒,𝑠𝑒𝑝}=3.44 \times  \mathrm{10^{19} m^{-3}}\ \ p_0^{0.31} R_0^{-0.5}
```  
We could in theory use this equation to derive $`p_0`$, the neutral pressure, given $`𝑛_{𝑒,𝑠𝑒𝑝}`$ which is already available in PROCESS.  However, this would require us to raise $`𝑛_{𝑒,𝑠𝑒𝑝}`$ to the power 1/0.31 = 3.2.  An error of a factor 1.5 in $`𝑛_{𝑒,𝑠𝑒𝑝}`$ would give a factor 3.7 error in $`P_{sep}`$ at detachment.  It would be much better to set $`p_0`$ as an iteration variable and use equation 6 as written to derive $`𝑛_{𝑒,𝑠𝑒𝑝}`$.  

We would need to add a constraint to ensure that $`𝑛_{𝑒,𝑠𝑒𝑝}`$ isn't too high compared to the pedestal density $`𝑛_{𝑒,ped}`$. 

Core Zeff and core Ar concentration:  These equations give $`Z_{eff}`$ and argon concentration in terms of the puffing rates for fuel and argon.  We don't need to use them, since these parameters are already available in PROCESS.

The electron temperature at the midplane separatrix, $`T_{𝑒,𝑠𝑒𝑝}`$, can be estimated using the 2-point model assuming that the target temperature is 0 eV (equation 7).  I don't think we need this equation, and I am not sure if it has been checked against SOLPS or experiments.

**Comment by jonmaddock** [comment]
In GitLab by @mn3981 on May 17, 2022, 16:15

created branch [`1647-surrogate-model-of-divertor-detachment`](/process/process/-/compare/develop...1647-surrogate-model-of-divertor-detachment) to address this issue

**Comment by jonmaddock** [comment]
In GitLab by @mkovari on May 18, 2022, 10:15

Hi Chris - I think creating a new branch may be jumping the gun a little.  We haven't really talked through the principles.

**Comment by jonmaddock** [comment]
In GitLab by @mn3981 on May 19, 2022, 13:28

Stuart Henderson comments (19/05/22):
- dpuff main driver on neutral pressure from SOLPS 
- Injecting argon decreased the neutral pressure 
- $`p_{0}`$ not on pump duct surface.
- Use an average value of 10 Pa for $`p_{0}`$ for detachment.
- Changing power will change separatrix temperature which will change separatrix density 
- Need fairly low separatrix density for EBW heating
- Stuart H has not fully investigated $`n_{e,sep}`$.
- 3-5 for enrichment factor. 
- Allowing $`c_{𝐴𝑟}^{core}`$ as an iteration variable
- $`q_{95}`$ term needed for connection length 
- Use $`\lambda_{int}`$ 
- No clear scaling for $`S`$ parameter
- **Model is not finished or checked** 
- **Terms need added to DP equation**

**Comment by mkovari** [comment]
An additional benefit is that it does not require any ADAS data.
